cirrus/server/cirrus/manifest_loader.py

```diff
@@ -0,0 +1,14 @@
+from typing import Any, Dict, List
+
+
+class ManifestLoader:
+    feature_manifest: List[Dict[str, Any]] = []
+
+    @classmethod
+    def get_latest_feature_manifest(cls) -> List[Dict[str, Any]]:
+        return cls.feature_manifest
+
+    @classmethod
+    def update_feature_manifest(cls,
+                                updated_feature_manifest: List[Dict[str, Any]]) -> None:
+        cls.feature_manifest = updated_feature_manifest
```

Comment:
```get_latest_feature_manifest```, does this imply there will be an "old" feature manifest?
```update_feature_manifest``` , does this imply you are updating the "latest" feature manifest or something else?

Reply:
latest will be an assumption that we are trying to get latest manifest copy which will be available as part of the container, and yes update_feature_manifest means we will be updating our copy of the latest feature manifest